item.rb

```diff
@@ -32,6 +32,11 @@ def move_to_archive
     @archived = true if can_be_archived?
   end
 
+  def add_author(author)
+    @author = author
+    @author.add_item(self) unless @author.items.include?(self)
+  end
+
```

Comment:
Hello @youmari , very nice job so far, seeing the author and add_author function, I can't seem to see much difference except the method name. I think we should refactor this code so as to ensure it meets the DRY principle of good code structure

Reply:
@Mosams  if you look at UML you'll see add_author method to implement the one to many relationship